qemu/tests/nic_teaming.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import time
+import re
+import random
+from virttest import utils_test, aexpect, utils_net
+from autotest.client.shared import error, utils
```

Comment:
import sequence:
1. build in modules first
2. autotest framework modules
3.  your custom modules.